facades/PC/build.gradle.kts

```diff
@@ -3,14 +3,15 @@
 
 // The PC facade is responsible for the primary distribution - a plain Java application runnable on PCs
 
+import Terasology_dist_gradle.ValidateZipDistribution
```

Comment:
That's not pretty.

We're using [precompiled script plugins](https://docs.gradle.org/current/userguide/custom_plugins.html#sec:precompiled_plugins) to cut down on the amount of config we have to use. It sets the gradle plugin name nicely, but this import path is gross. I wonder if that's something we can change without giving up its convenience.

Or is there a way we can have our Task class auto-imported like the core gradle Tasks seem to be?

Reply:
hmm. not seeing much to do about this. I can put a namespace in front of it like `org.terasology.foo.` but it still involves that `_gradle` part.

I'm inclined to leave it be until we get to the point where we're packaging our gradle plugins for distribution for use outside this multiproject.

Reply:
try [`@JvmName`](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.jvm/-jvm-name/)

Reply:
hmm, will try later, not worth changing generated filenames at this point in the PR, as we already have another PR waiting on this one.